modules/ossm-config-sec-mtls-mesh.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-config.adoc
+
+[id="ossm-security-enabling-strict-mtls_{context}"]
+= Enabling strict mTLS across the service mesh
+
+If your workloads do not communicate with outside services, you can quickly enable mTLS across your mesh without commutation interruptions. You can enable it by setting `spec.security.dataPlane.mtls` to `true` in the `ServiceMeshControlPlane` resource. The Operator creates the required resources.
+
```

Comment:
commutation?

Reply:
@knrc Good catch. I made that change. Ready for another look.